src/probability/probabilityScript.sml

```diff
@@ -1,3 +1,4 @@
+(* -*- Mode: Holscript; -*- *)
```

Comment:
Please delete this line; if you're using emacs, you should set up your editor configuration to automatically recognise `*Script.sml` as files that should use `holscript-mode`.

Reply:
@mn200 I got this. And already updated code.